common/src/main/java/io/netty/util/internal/NativeLibraryLoader.java

```diff
@@ -27,13 +27,16 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.OutputStream;
+import java.io.UncheckedIOException;
```

Comment:
unused ?

Reply:
Yes. It is JDK 1.8+ and I replaced it with RuntimeException. Removing it!

Reply:
Removed!